### ISSUE TITLE
vspd support

### ISSUE THREAD

**Issue by jholdstock** [decred/dcrwebapi#98]
dcrwebapi currently has a [method named gsd](https://api.decred.org/?c=gsd), which is used to retrieve a list of known dcrstakepool instances. This is used to populate the VSP list in decrediton, on the dcrstakepool front page, and decrediton.

Each VSP in the response has the following fields:
- `APIVersionsSupported`, referring to the supported versions of the dcrstakepool API.
- `UserCount` and `UserCountActive` referring to registered users.
- `Version` referring to the dcrstakepool version.

I suggest that none of these fields are really applicable to vspd, and that adding vspd into this API response makes little sense. We should probably add a new method to dcrwebapi which is specifically for returning vspd instances.

Once dcrstakepool is fully deprecated and support is removed from dcrwallet/decrediton, the old gsd call can be removed from the API.

VSP operators may choose to run both dcrstakepool and vspd, so there may be some overlap in the data returned by the old call and the new one.

**Comment by vctt94** [comment]
Agree on that!

**Comment by jholdstock** [comment]
Rough idea for fields to included in the new call

```go
APIVersions          []int64
Network              string 
Launched             int64
LastUpdated          int64
FeePercentage        float64
Closed               bool   
Voting               int64  
Voted                int64  
Revoked              int64  
Version              string
```

This will depend on getting decred/dcrwallet#1783 merged, followed by decred/vspd#86

An map keyed with URL would yield JSON like this:

```json
{
	"https://teststakepool.decred.org":{
		"apiversions": [1],
		"network": "testnet",
		"launched": 1513878600,
		"lastupdated": 1594292788,
		"feepercentage": 2.5,
		"closed": false,
		"voting": 42,
		"voted": 747,
		"revoked": 3,
		"version": "1.0.0"
	},

	"https://myvsp.com":{
		...
	},

	...
}
```
